ffmuc-autoupgrade-experimental2testing/files/lib/gluon/upgrade/505-move-experimental2testing

```diff
@@ -1,11 +0,0 @@
-#!/usr/bin/lua
-
-local uci = require("simple-uci").cursor()
-
-local branch = uci:get('autoupdater', 'settings', 'branch')
-
-if "experimental" == branch then
-    uci:set('autoupdater', 'settings', 'branch', 'testing')
```

Comment:
I would like to "keep" this package for future reference.
Context: We have the stale wireguard branch which I would like to migrate to experimental via this approach (and there might be next we might want to use this code with in future as well)

Reply:
Resolved